tests/lax_numpy_test.py

```diff
@@ -235,11 +233,11 @@ def op_record(name, nargs, dtypes, shapes, rng_factory, diff_modes,
     op_record("expm1", 1, number_dtypes, all_shapes, jtu.rand_small_positive,
               [], tolerance={np.float64: 1e-8}, inexact=True),
     op_record("fix", 1, float_dtypes, all_shapes, jtu.rand_default, []),
-    op_record("fix", 1, int_dtypes + unsigned_dtypes, all_shapes,
+    op_record("fix", 1, int_dtypes + unsigned_dtypes_no64, all_shapes,
               jtu.rand_default, [], check_dtypes=False),
     op_record("floor_divide", 2, float_dtypes + int_dtypes,
               all_shapes, jtu.rand_nonzero, ["rev"]),
-    op_record("floor_divide", 2, unsigned_dtypes, all_shapes,
+    op_record("floor_divide", 2, unsigned_dtypes_no64, all_shapes,
```

Comment:
This does remove a useful test case, which is u64 // u64. Same for most of the binary operators changed by this PR.

I wonder if these test harness would be better expressed by providing the complete set of valid types for all arguments, built using some helpers.

Another option would be to add another record just for the u64/u64 case.

(Not to mention: we should probably be testing invalid types also and checking for errors.)